helm-charts/orchestrator/templates/sonataflows.yaml

```diff
@@ -32,6 +32,15 @@ spec:
         limits:
           memory: {{ .Values.orchestrator.sonataflowPlatform.resources.limits.memory }}
           cpu: {{ .Values.orchestrator.sonataflowPlatform.resources.limits.cpu }}
+  {{- if (and (.Values.orchestrator.sonataflowPlatform.broker.name) (.Values.orchestrator.sonataflowPlatform.broker.name)) }}
```

Comment:
perhaps you mean `Values.orchestrator.sonataflowPlatform.broker.namespace` as well as `Values.orchestrator.sonataflowPlatform.broker.name` ?

Reply:
yes, thx for noticing the error :)